ci-jobs/templates/package.yml

```diff
@@ -21,9 +21,9 @@ jobs:
     steps:
       - checkout: self
         persistCredentials: true
-      - task: NodeTool@0
+      - task: UseNode@1
```

Comment:
consider using lts  node version alias instead of the hardcoded one

Reply:
I'm not sure if Azure supports the `lts` value. [The documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-node-v1?view=azure-pipelines#inputs) only refers to semver values.

Reply:
Oh, I forgot it's Azure. Perhaps it makes sense to migrate to GitHub CI

Reply:
I agree that GitHub CI would be more convenient, though I'm guessing there was a reason why Azure was originally selected. Maybe the machines are more powerful? It's also used in other Appium projects (e.g. xcuitest driver).

Reply:
it was selected before github had any CI. Yes, it is still used, although not by xcuitest driver. It's used in webdriveragent because of the high amount of e2e tests we run there and the limited capability of free macOS-based runners on Github. Otherwise I would also migrate them.

Reply:
Perhaps I tried to convert this project to GHA before. Then, I needed maybe Window's signature which probably needed Dan's help. We could restart migrating them.